src/query/runner/ipa.rs

```diff
@@ -67,7 +67,7 @@ where
         DowngradeMalicious<Target = MCAggregateCreditOutputRow<F, AdditiveShare<F>, BreakdownKey>>,
     AdditiveShare<F>: Serializable,
 {
-    #[tracing::instrument("ipa_query", skip_all)]
+    #[tracing::instrument("ipa_query", skip_all, fields(query_config=%self.config, %query_size))]
```

Comment:
```suggestion
    #[tracing::instrument("ipa_query", skip_all, fields(query_config=?self.config, %query_size))]
```

given that `Display` implementation outputs the `Debug`, we can ask tracing to do it for us. I am not sure however how verbose it is, we may still need a specialized `Display` or customized `Debug` to make it more concise